src/style.css

```diff
@@ -1,3 +1,63 @@
+html {
+  --head-marg: 50px;
+  --special-color: #00acdc;
+}
+
 body {
+  font-family: Roboto, sans-serif;
```

Comment:
You have only one font specified. It's recommended to have a fallback font in case the main one doesn't load. For example, you could have `font-family: Roboto, Arial, sans-serif;`.